src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic.cs

```diff
@@ -1,22 +1,12 @@
-#pragma warning disable IDE0073
-//
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-//
-#pragma warning restore IDE0073
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is a verbatim copy of https://github.com/microsoft/msquic/blob/main/src/cs/lib/msquic.cs . These changes will be overwritten by the next msquic interop update.

Reply:
Same comment for all files in this directory.